GOVERNANCE.md

```diff
@@ -0,0 +1,191 @@
+
+Reviewers have all the rights and responsibilities of a Contributor, plus:
+
+* Responsibilities include:
+    * Regular contribution of pull requests to the Podman project or its subprojects
+    * Triage of Github issues on the Podman project or its subprojects
+    * Regularly fixing Github issues on the Podman project or its subprojects
+    * Following the [reviewing guide](./REVIEWING.md) and [issue triage guide](./TRIAGE.md)
+    * A sustained high level of pull request reviews on the Podman project or one of its subprojects
+    * Assisting new Contributors in their interactions with the project
+    * Helping other contributors become reviewers
+* Requirements:
+    * Has a proven record of good-faith contributions to the project as a Contributor for a period of at least 6 months. The time requirement may be overridden by a supermajority (66%) vote of Maintainers and Core Maintainers.
+    * Has participated in pull request review and/or issue triage on the project for at least 6 months. The time requirement may be overridden by a supermajority (66%) vote of Maintainers and Core Maintainers.
+    * Is supportive of new and occasional contributors and helps get useful PRs in shape to merge
+* Additional privileges:
+    * Has rights to approve pull requests in the Podman project or a subproject, marking them as ready for a Maintainer to review and merge
+    * Can recommend and review other contributors to become Reviewers
+
+In repositories using an OWNERS file, Reviewers are listed as Reviewers in that file.
+
+#### The process of becoming a Reviewer is:
+1. The contributor must be sponsored by a Maintainer. That sponsor will open a PR against the appropriate repository, which adds the nominee to the MAINTAINERS.md file as a reviewer.
+2. The contributor will add a comment to the pull request indicating their willingness to assume the responsibilities of a Reviewer.
+3. At least two Maintainers of the repository must concur to merge the PR.
+
+### Maintainer
+Description: Maintainers are established contributors with deep technical knowledge of the Podman project and/or one of its subprojects. Maintainers are granted the authority to merge pull requests, and are expected to participate in making decisions about the strategy and priorities of the project. Maintainers are responsible for code review and merging in a single repository or subproject. It is possible to become Maintainer of additional repositories or subprojects, but each additional repository or project will require a separate application and vote. They are able to participate in all maintainer activities, including Core Maintainer meetings, but do not have a vote at Core Maintainer meetings.
+
+In repositories using an OWNERS file, Maintainers are listed as Approvers in that file.
```

Comment:
If this is not a common file to be shared, I'd include links to things like the OWNERS file.  If it will be common to more than one project, ignore this.

Reply:
This will be referred to by Buildah, Podman, and Skopeo repos.